code/datums/gamemode/dynamic/dynamic_rulesets_latejoin.dm

```diff
@@ -73,3 +73,34 @@
 	federation.HandleRecruitedRole(newWizard)
 	newWizard.Greet(GREET_LATEJOIN)
 	return 1
+
+//////////////////////////////////////////////
+//                                          //
+//            AI MALFUNCTION                ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+//                                          //
+//////////////////////////////////////////////
+
+/datum/dynamic_ruleset/latejoin/malfunction
+	name = "Malfunctioning AI"
+	role_category = ROLE_MALF
+	enemy_jobs = list("Security Officer","Detective", "Warden", "Head of Security", "Captain")
+	exclusive_to_jobs = list("AI")
+	required_enemies = list(4,4,4,4,4,4,2,2,2,0)
+	required_candidates = 1
+	weight = 3
+	cost = 35
+	requirements = list(90,90,90,90,80,70,50,30,20,10)
```

Comment:
This isn't a criticism exactly but just posting for the help of other people in discussion. With this threat requirement bracket, AI Malfunction will probably never fire under 35 players. Floating 50 threat doesn't happen very often, as it usually gets spent on other antags.

Reply:
They're copied from the roundstart ruleset, I don't really care one way or the other.

Reply:
It's the same reason you never see malf AI anymore as a round type, which people have commented on. Granted, my experience is anecdotal, but I observe a lot of rounds.

And again this isn't necessarily to say I think it needs to be changed, merely providing some background knowledge in case people want to discuss rates.